browser-test/src/applicant/northstar_upsell.test.ts

```diff
@@ -206,6 +259,19 @@ test.describe('Upsell tests', {tag: ['@northstar']}, () => {
     })
   }
 
+  async function validateApplicationDownloadLink(
+    page: Page,
+    expectApplicationDownloadLink: boolean,
+  ) {
+    await test.step('Validate application submitted page', async () => {
```

Comment:
Nit: Should this be something like 'Validate download link?'

Reply:
fixed